use_cases/eluc/predictors/evaluator.py

```diff
@@ -0,0 +1,26 @@
+from sklearn.metrics import mean_absolute_error
+
+from data.eluc_data import ELUCData
+from predictors.predictor import Predictor
+
+class Evaluator():
```

Comment:
Very simple evaluator. Holds a test set in memory and then can evaluate a given Predictor object on it. This way, if we end up hiding the test set this is where we can download it.